### ISSUE TITLE
Tasks 22/01/2023

### ISSUE THREAD

**Issue by mmattamala** [leggedrobotics/wild_visual_navigation#197]
## Integration tasks
- [x] **Review, commit and push** changes in OPC, NPC, LPC, Jetson
- [x] **Copy and clean** bags from robot
- [x] **Copy WVN data output**
- [x] Check if there are missing parameters in the rosbag recording (twist sent to locomotion controller
- [x] Check integration of `wide_angle_camera` as input (high priority) - potentially design a hack for the calibration available in the camera_info topic. Look for documentation in the ANYmal manual
- [ ] Check integration of multiple sources as input for training (low priority, depends on the previous one)
- [x] Define better parameters for local planner (thresholds for goal, max speed)
- [ ] Enable option to reset learning, dumping the data to the mission and making a new mission path (low priority, we are not using this feature)
- [ ] Fix how the images are processed when multiple inputs are set
       Currently, it processes the images in a messy way as it is not following any specific order
- [ ] Setting up Orin 

## Other visualizations
- [ ] Inspect the data produced by WVN after each mission
- [ ] Generate plotting tools for appendix

## Ablation strategy
- [x] Ablate the selection of the std_loss hyperparameter and false_positive impact.
- [ ] Generate confidence score for all experiments (Finish running the timing / data ablation)
- [ ] (Small things) Temporal consistency (Maybe check if the implementation is good)
- [ ] (Small things) Pre-trained model; Add pyramid pooling layer to the network if the dimension is too large or check what other people did for this.
- [x] Convert the results into a table using pandas dataframe in notebook.
- [ ] Create the plots needed for examples on a matrix showing for example the output of different settings (Supplementary).

## Organization
- [ ] Check RSS submission webpage -> [**no news so far**](https://roboticsconference.org/information/cfp/)
- [ ] Resume paper writing

## Final decision on experiments
- [ ] (Showing Adaptation) Walking in a loop on grass and path in the Oxford university campus
    - Showing the evolution of the system. Images, Traversability Maps, Uncertainty changes.
- [ ] (Closed Loop) Preference to walk on a hiking path - comparison to geometry walking straight
- [ ] (Closed Loop) High grass - comparison to geometry failing
- [ ] (Continous Supervision Signal) Showing robot struggling / predicting the struggle. Showing maybe a plot on a top down view map with the different color coded trajectories and the traversability signal (Velocity/Binary)

## Ablations Results
- Lesson LR: Keep adam 0.001 best for everything right choice
-  Lesson Confidence - No clear results depends on the dataset and does not have a major impact 
- Dino performs clearly better than the other networks use dino 224_16 given also compute needed
- The network most likely go with large MLP given best performance across scenes
- Classical learning is worse
- Loss function and anomaly
- 
# Ablation Tasks:
- [ ] Figure out what is going on with the loss.
